app/routes/index.js

```diff
@@ -0,0 +1,25 @@
+// internal modules
+const fs = require('fs');
+
+// 3rd party modules
+const logger = require('winston');
+
+function registerRoutes(app)
+{
+  logger.info("Add Routers..");
+  fs.readdirSync(__dirname).forEach(function (file) {
+    if (!file.match(/index\.js$/) &&
+         file.match(/\.js$/) &&
+        !file.match(/error\.js$/)) {
+      logger.verbose(' * '+file);
+      require(`${__dirname}/${file}`)(app);
+    }
+  });
+}
+module.exports.registerRoutes = registerRoutes;
+
+function registerErrorRoute(app)
+{
```

Comment:
brackets for functions should start from the same line they are introduced
eg. `function foo() {`